src/plugins/auto_batch/auto_batch.cpp

```diff
@@ -841,7 +841,7 @@ InferenceEngine::IExecutableNetworkInternal::Ptr AutoBatchInferencePlugin::LoadN
         // find the batch dim
         ov::pass::Manager m;
         m.register_pass<ngraph::pass::InitNodeInfo>();
-        m.register_pass<ov::pass::FindBatch>(true, check_dims);
+        m.register_pass<ov::pass::FindBatch>(false, check_dims);
```

Comment:
the DO is already removed by the HETERO, so this "true" had no effect anyway, but better to avoid it